app/lib/matador/components/job-info/JobInfo.tsx

```diff
@@ -0,0 +1,21 @@
+import { Grid, MantineColor } from '@mantine/core';
+import { StatCard } from '../stat-card';
+import JobInfoProps from './JobInfoProps';
+
+
+const JobInfo = ({ infoName, infoValue, color } : JobInfoProps) => {
+
+    const colors : MantineColor[] = ['blue', 'grape', 'indigo', 'green', 'red', 'cyan', 'orange']
+
+    return (
+        <Grid.Col span={12}>
+            <StatCard
+                title={infoName}
+                value={infoValue}
+                color={color ?? colors[Math.floor(Math.random()*colors.length)]}
+            />
+        </Grid.Col>
+    );
+}
+
+export default JobInfo;
```

Comment:
Why this is being exported as default?

Reply:
I like to do this:
- Exporting the component as default, so if I need the component this is the import statement: `import Component from './Component'`

- If I need the props of that component, the import statement is : `import Component, { ComponentProps } from './Component'`

It's my way to style component imports

Reply:
Ok, I like it.

But so, I would ask you to refactor all components that have been declared as anonymous function in the following way:

```javascript
export default function MyComponent() {}
```

We need this because naming a function helps us in case of errors, if we use anonymous function, we can not see the full strack trace, making it harder to debug

Reply:
But when I have an error in the ui components, I am pretty sure that I can see all the stack traces

Try to read this https://stackoverflow.com/a/48764221.